bin/cli.js

```diff
@@ -473,6 +473,22 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					const statsString = stats.toString(outputOptions);
 					const delimiter = outputOptions.buildDelimiter ? `${outputOptions.buildDelimiter}\n` : "";
 					if (statsString) stdout.write(`${statsString}\n${delimiter}`);
+
+
+					const now = new Date();
+					const MONDAY = 1;
+					const SIX_DAYS = 518400000;
+					const openCollectivePath = __dirname + "/opencollective.js";
+					if (now.getDay() === MONDAY) {
+						const { statSync, utimesSync } = require("fs");
+						const lastPrint = statSync(openCollectivePath).atime;
+						const lastPrintTS = new Date(lastPrint).getTime();
+						if (now.getTime() - lastPrintTS > SIX_DAYS) {
```

Comment:
abstract to expression or function that has the naming `installedLaterThanSixDays` or similar.

Reply:
I've abstracted to a variable:

```js
const timeSinceLastPrint = now.getTime() - lastPrintTS;
if (timeSinceLastPrint > SIX_DAYS) {
```

Thanks for your comments, let me know if I can make any other improvements 👍